<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectPopup</class>
 <widget class="QWidget" name="connectPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>621</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="currentCountNumberLabel">
      <property name="text">
       <string>Current Points: 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ADCValueLabel">
      <property name="text">
       <string>AD Value:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="currentWeightTextLabel">
      <property name="text">
       <string>Current Weight</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="weightBox">
      <property name="maximum">
       <double>9999999999.989999771118164</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="unitBox">
      <item>
       <property name="text">
        <string>kg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>g</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="setPointButton">
      <property name="text">
       <string>Set Point</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextPointButton">
      <property name="text">
       <string>Next Point</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calibrateButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>203</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="CANIdbox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CAN ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
